### PR TITLE
Pokemon spite fixes

### PR DESCRIPTION
**Pull request by cafei-uh**
Numerous fixes/improvements to different Pokemon sprites.
## Images
**Sneasler**: Improved shiny follower; added green eyes to shiny front.
image -> image
image
**Hisuian Zorua**: Removed a stray pixel.
image

**Galarian Yamask**: Added shiny follower palette; added missing purple to shiny front.
image image

**Runerigus**: Added missing cyan eye to its shiny front.
image

**Galarian Darumaka**: Added shiny follower palette; improved shiny front palette.
image image

**Galarian Zigzagoon**: Added shiny follower palette; improved side walking cycle.
image

**Cursola, Pincurchin**: Fixed shiny front.
image image

**Galarian Zapdos, Hisuian Samurott, Sawsbuck Summer, Hisuian Sneasel, Hisuian Quilfish, Morelull, Bounsweet, Bruxish, Guzzlord, Regieleki**: improved/fixed shiny follower palette.
image image image image image image image image image image

**Galarian Ponyta**: Added shiny follower palette; slightly improved follower sprite.
image

**Galarian Rapidash, Galarian Slowpoke, Galarian Farfetchd, Galarian Mr. Mime, Galarian Weezing, Galarian Articuno, Galarian Moltres, Galarian Slowking, Galarian Stunfisk, Galarian Darmanitan, Crowned Zacian + Zamazenta**: Added shiny follower palette.
image image  image image  image image image  image image image image image

### CODE REVIEW

File: graphics/pokemon/sneasel/hisuian/overworld_shiny.pal
Comment (hedara90) [inline graphics/pokemon/sneasel/hisuian/overworld_shiny.pal:1]: I think that the tail plume should still be visibly yellow/gold, not just the browner shade. But that requires tweaking of the sprite.

File: graphics/pokemon/bruxish/overworld_shiny.pal
Comment (hedara90) [inline graphics/pokemon/bruxish/overworld_shiny.pal:1]: Stray green pixel on lips on south sprite.

File: graphics/pokemon/regieleki/overworld_normal.pal
Comment (hedara90) [inline graphics/pokemon/regieleki/overworld_normal.pal:1]: Not your fault, but the eye pattern is wrong on the follower sprite, should be "X", not "Y"

File: graphics/pokemon/farfetchd/galarian/overworld_shiny.pal
Comment (hedara90) [inline graphics/pokemon/farfetchd/galarian/overworld_shiny.pal:1]: Bottom of leek on south follower sprite changes color between normal and shiny where it shouldn't.

File: graphics/pokemon/weezing/galarian/overworld_shiny.pal
Comment (hedara90) [inline graphics/pokemon/weezing/galarian/overworld_shiny.pal:1]: Outline on Weezing chimney smoke changes color where it shouldn't from normal to shiny follower sprite.

File: graphics/pokemon/articuno/galarian/overworld_shiny.pal
Comment (hedara90) [inline graphics/pokemon/articuno/galarian/overworld_shiny.pal:1]: Primary feathers could be a bit whiter.

File: graphics/pokemon/slowking/galarian/overworld_shiny.pal
Comment (hedara90) [inline graphics/pokemon/slowking/galarian/overworld_shiny.pal:1]: Mantle should change shade of purple.

**Review by hedara90 — COMMENTED** [review]
Looking good, just a few comments/suggestions if they're workable.
You knocked quite a few items off my list of sprites to fix.

File: graphics/pokemon/sneasel/hisuian/overworld_shiny.pal
Comment (cafei-uh) [reply graphics/pokemon/sneasel/hisuian/overworld_shiny.pal:1]: image
Copied Johtonian Sneasel`s tail

File: graphics/pokemon/bruxish/overworld_shiny.pal
Comment (cafei-uh) [reply graphics/pokemon/bruxish/overworld_shiny.pal:1]: image

File: graphics/pokemon/regieleki/overworld_normal.pal
Comment (cafei-uh) [reply graphics/pokemon/regieleki/overworld_normal.pal:1]: image
image

File: graphics/pokemon/farfetchd/galarian/overworld_shiny.pal
Comment (cafei-uh) [reply graphics/pokemon/farfetchd/galarian/overworld_shiny.pal:1]: image / image
image / image

File: graphics/pokemon/weezing/galarian/overworld_shiny.pal
Comment (cafei-uh) [reply graphics/pokemon/weezing/galarian/overworld_shiny.pal:1]: image

File: graphics/pokemon/articuno/galarian/overworld_shiny.pal
Comment (cafei-uh) [reply graphics/pokemon/articuno/galarian/overworld_shiny.pal:1]: image
Also removed weird blue feather on normal too and made white feather more prominent image

File: graphics/pokemon/slowking/galarian/overworld_shiny.pal
Comment (cafei-uh) [reply graphics/pokemon/slowking/galarian/overworld_shiny.pal:1]: image
Had to change some to make it possible. Also just noticed that front's shiny doesn't change the mantle, should be fixed later as well

**Comment by cafei-uh** [comment]
Thanks for the review! Addressed everything and ready for a rereview

**Review by hedara90 — APPROVED** [review]
Looks good.

**Pull request merged**